gulp/tasks/browserify.js

```diff
@@ -59,18 +60,23 @@ function buildScript(file) {
 
         gutil.log('Rebundle...');
 
-        return stream.on('error', handleErrors)
-            .pipe(source(file))
-            .pipe(gulpif(createSourcemap, buffer()))
-            .pipe(gulpif(createSourcemap, sourcemaps.init()))
-            .pipe(gulpif(global.isProd, streamify(
-                uglify({
-                    compress: { drop_console: true }
-                })
-            )))
-            .pipe(gulpif(createSourcemap, sourcemaps.write('./')))
-            .pipe(gulp.dest(config.scripts.dest))
-            .pipe(gulpif(browserSync.active, browserSync.reload({ stream: true, once: true })));
+        return stream.on('error', function() {
```

Comment:
Should this be `function(handleErrors)`?

Reply:
That occurred to me after reading the example at https://github.com/terinjokes/gulp-uglify/tree/master/docs/why-use-pump

Reply:
Out of interest, why has this anonymous callback replaced the error handler in the original code?